spec/process_booking_spec.rb

```diff
@@ -50,7 +50,14 @@
     fill_in 'Reference number', with: '12345678'
 
     click_button 'Send email'
+
+    # The step below the sleep fails interminently, attempt to fix this with a
+    # hard sleep as I have already increased the retries from 10s to 20s.
+    sleep(1)
     confirmation_email = retry_for(20, ->(email) { email }) {
+      # Log how many messages the API is returning to help debugging intermitent
+      # issues.
+      STDOUT.puts "Matched messages: #{Mailtrap.instance.search_messages(visitor.email)}"
```

Comment:
Yes, I was aware of it. Don't think it will make a difference, will mean that we can delete the debugging easily when done